Doc/reference/datamodel.rst

```diff
@@ -2376,10 +2378,9 @@ left undefined.
 
    .. note::
 
-      If the right operand's type is a subclass of the left operand's type and that
-      subclass provides the reflected method for the operation, this method will be
-      called before the left operand's non-reflected method.  This behavior allows
-      subclasses to override their ancestors' operations.
+      If the right operand's type is a subclass of the left operand's type, the
+      reflected method having precedence allows subclasses to override their
+      ancestors' operations.
 
```

Comment:
This new text is fine, but make it a foot note like the others instead of an in-line note.